include/gbt.h

```diff
@@ -11,96 +11,67 @@
 #include "utils.h"
 
 /*! \brief Scan the GBT phases of one OptoHybrid.
- *  \param[in] request RPC response message
- *  \param[out] response RPC response message
- *
- *  The method expects the following RPC keys :
- *  - `word ohN` : OptoHybrid index number.
- *  - `word N` : The number of times the scan must performed.
- *  - `word phaseMin` : Lowest phase to scan (min = 0).
- *  - `word phaseMax` : Highest phase to scan (max = 15).
- *  - `word phaseStep` : Step to scan the phases.
- *
- *  The method returns the following RPC keys :
- *  - `word_array OHX.VFATY` : Array of word containing the results of the scans. See details of the `scanGBTPhase` function for a description of the words.
- *  - `string error` : If an error occurs, this keys exists and contains the error message.
- */
-void scanGBTPhases(const RPCMsg *request, RPCMsg *response);
-
-/*! \brief Local callable version of `scanGBTPhase`.
- *  \param[in, out] la Local arguments structure.
  *  \param[in] ohN OptoHybrid index number.
  *  \param[in] N The number of times the scan must performed.
  *  \param[in] phaseMin Lowest phase to scan (min = 0).
  *  \param[in] phaseMax Highest phase to scan (max = 15).
  *  \param[in] phaseStep Step to scan the phases.
- *  \return Returns `false` in case of success; `true` in case of error. The precise error is logged and written to the `error` RPC key.
+ *  \return Returns a 2-dimensional array containing each phase of each VFAT in each OptoHybrid. The precise error is logged and written to the `LOG4CPLUS` logger.
```

Comment:
It's not for *each* `OptoHybrid`, it's only for the `OptoHybrid` specified by `ohN`

Reply:
Also, no need to mention about the error logging, rather mention *if* it throws any exceptions, and if so, which, and under what conditions

Reply:
Unless there is extra info, explicit logging is redundant since `xhal` [will likely get this feature](https://github.com/cms-gem-daq-project/xhal/blob/feature/templated-rpc-methods/xhalcore/include/xhal/rpc/register.h#L34).